stable/jenkins/templates/config.yaml

```diff
@@ -155,6 +155,20 @@ data:
       <pendingClasspathEntries/>
     </scriptApproval>
 {{- end }}
+  jenkins.model.JenkinsLocationConfiguration.xml: |-
+    <?xml version='1.1' encoding='UTF-8'?>
+    <jenkins.model.JenkinsLocationConfiguration>
+      <adminAddress>{{ default "nobody@nowhere.io" .Values.Master.JenkinsAdminEmail }}</adminAddress>
+{{- if .Values.Master.HostName }}
+{{- if .Values.Master.JenkinsUriSecure }}
```

Comment:
Where is this coming from? Could we use `Master.Ingress.TLS` instead?

Reply:
You are totally right. There is no instance I can think of where I would not configure TLS on the ingress and want an https:// uri.